### DISCUSSION TITLE
R68S 7.19版本，ssrplus更新订阅链接失败，把所有节点丢弃了

### DISCUSSION THREAD

**Discussion by kistuvze** [coolsnowwolf/lede#9828] [R68S / 电犀牛专区]
![捕获](https://user-images.githubusercontent.com/109887923/180632414-c87d413b-f32b-489d-847e-0b48e8a4e589.PNG)

**Comment by kistuvze** [comment]
在v2rayN客户端可以正常订阅和使用节点，ssrplus更新订阅后结点列表里没有节点，在日志中发现节点全被丢弃了

**Comment by EnnawYang** [comment]
https://github.com/fw876/helloworld/issues/933

**Reply by kistuvze** [reply]
谢谢
